src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,44 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        Direction directToX;
+        Direction directToY;
```

Comment:
do we need these variables? 
let's remove them)

Reply:
done